spec/views/emails/template.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe "emails/template", type: :view do
+  before(:each) do
+    sign_in(FactoryGirl.create(:user, role: :admin))
+
+    @template = FactoryGirl.create(:email_template)
+    @templates = Array(@template)
+    render :partial => "emails/templates"
+  end
+
+  it ("renders template headline") do
```

Comment:
I think we never used () around the "" in tests (or at least I havent written or seen this before)